### PR TITLE
[zh] sync /working-with-objects/_index.md annotations field-selectors

### PR DESCRIPTION
**Pull request by asa3311**

### CODE REVIEW

File: content/zh-cn/docs/concepts/overview/working-with-objects/_index.md [hunk 1]
```diff
@@ -139,32 +139,36 @@ For more information on the object spec, status, and metadata, see the
...
 ### 描述 Kubernetes 对象    {#describing-a-kubernetes-object}
 
 创建 Kubernetes 对象时，必须提供对象的 `spec`，用来描述该对象的期望状态，
 以及关于对象的一些基本信息（例如名称）。
 当使用 Kubernetes API 创建对象时（直接创建或经由 `kubectl` 创建），
 API 请求必须在请求主体中包含 JSON 格式的信息。
-**大多数情况下，你需要提供 `.yaml` 文件为 kubectl 提供这些信息**。
-`kubectl` 在发起 API 请求时，将这些信息转换成 JSON 格式。
-
+大多数情况下，你需要提供 manifest 文件为 `kubectl` 提供这些信息。
```
Comment (howieyuen) [inline content/zh-cn/docs/concepts/overview/working-with-objects/_index.md:154]:
```suggestion
大多数情况下，你需要提供 _manifest_ 文件为 `kubectl` 提供这些信息。
```

File: content/zh-cn/docs/concepts/overview/working-with-objects/field-selectors.md [hunk 2]
```diff
@@ -78,6 +78,15 @@ You can use the `=`, `==`, and `!=` operators with field selectors (`=` and `==`
 kubectl get services  --all-namespaces --field-selector metadata.namespace!=default
 ```
 
+{{< note >}}
+<!--
+[Set-based operators](/docs/concepts/overview/working-with-objects/labels/#set-based-requirement)
+(`in`, `notin`, `exists`) are not supported for field selectors.
+-->
+[基于集合的操作符](/zh-cn/docs/concepts/overview/working-with-objects/labels/#set-based-requirement)
+（`in`、`notin`、`exists`）不支持字段选择器。
```
Comment (howieyuen) [inline content/zh-cn/docs/concepts/overview/working-with-objects/field-selectors.md:87]:
```suggestion
（`in`、`notin`、`exists`）不支持字段选择算符。
```

**Review by howieyuen — CHANGES_REQUESTED** [review]

File: content/zh-cn/docs/concepts/overview/working-with-objects/_index.md [hunk 3]
```diff
@@ -139,32 +139,36 @@ For more information on the object spec, status, and metadata, see the
...
 ### 描述 Kubernetes 对象    {#describing-a-kubernetes-object}
 
 创建 Kubernetes 对象时，必须提供对象的 `spec`，用来描述该对象的期望状态，
 以及关于对象的一些基本信息（例如名称）。
 当使用 Kubernetes API 创建对象时（直接创建或经由 `kubectl` 创建），
 API 请求必须在请求主体中包含 JSON 格式的信息。
-**大多数情况下，你需要提供 `.yaml` 文件为 kubectl 提供这些信息**。
-`kubectl` 在发起 API 请求时，将这些信息转换成 JSON 格式。
-
+大多数情况下，你需要提供 _manifest_ 文件为 `kubectl` 提供这些信息。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/overview/working-with-objects/_index.md:154]:
```suggestion
大多数情况下，你会通过 **清单（Manifest)** 文件为 `kubectl` 提供这些信息。
```

File: content/zh-cn/docs/concepts/overview/working-with-objects/_index.md [hunk 4]
```diff
@@ -139,32 +139,36 @@ For more information on the object spec, status, and metadata, see the
...
+按照惯例，manifest 是 YAML 格式的（你也可以使用 JSON 格式）。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/overview/working-with-objects/_index.md:155]:
```suggestion
按照惯例，清单是 YAML 格式的（你也可以使用 JSON 格式）。
```

File: content/zh-cn/docs/concepts/overview/working-with-objects/_index.md [hunk 5]
```diff
@@ -139,32 +139,36 @@ For more information on the object spec, status, and metadata, see the
...
+像 `kubectl` 这样的工具在通过 HTTP 进行 API 请求时，
+会将 manifest 中的信息转换为 JSON 或其他受支持的序列化格式。
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/overview/working-with-objects/_index.md:157]:
```suggestion
会将清单中的信息转换为 JSON 或其他受支持的序列化格式。
```

File: content/zh-cn/docs/concepts/overview/working-with-objects/_index.md [hunk 6]
```diff
@@ -139,32 +139,36 @@ For more information on the object spec, status, and metadata, see the
...
 <!--
-Here's an example `.yaml` file that shows the required fields and object spec for a Kubernetes Deployment:
+Here's an example manifest that shows the required fields and object spec for a Kubernetes
+Deployment:
 -->
-这里有一个 `.yaml` 示例文件，展示了 Kubernetes Deployment 的必需字段和对象 `spec`：
+这里有一个 manifest 示例文件，展示了 Kubernetes Deployment 的必需字段和对象 `spec`：
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/overview/working-with-objects/_index.md:162]:
```suggestion
这里有一个清单示例文件，展示了 Kubernetes Deployment 的必需字段和对象 `spec`：
```

File: content/zh-cn/docs/concepts/overview/working-with-objects/_index.md [hunk 7]
```diff
@@ -139,32 +139,36 @@ For more information on the object spec, status, and metadata, see the
...
+{{% code_sample file="application/deployment.yaml" %}}
 
 <!--
-One way to create a Deployment using a `.yaml` file like the one above is to use the
+One way to create a Deployment using a manifest file like the one above is to use the
 [`kubectl apply`](/docs/reference/generated/kubectl/kubectl-commands#apply) command
 in the `kubectl` command-line interface, passing the `.yaml` file as an argument. Here's an example:
 -->
-相较于上面使用 `.yaml` 文件来创建 Deployment，另一种类似的方式是使用 `kubectl` 命令行接口（CLI）中的
+相较于上面使用 manifest 文件来创建 Deployment，另一种类似的方式是使用 `kubectl` 命令行接口（CLI）中的
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/overview/working-with-objects/_index.md:171]:
```suggestion
与上面使用清单文件来创建 Deployment 类似，另一种方式是使用 `kubectl` 命令行接口（CLI）的
```

File: content/zh-cn/docs/concepts/overview/working-with-objects/_index.md [hunk 8]
```diff
@@ -193,7 +198,7 @@ In the `.yaml` file for the Kubernetes object you want to create, you'll need to
 -->
 ### 必需字段    {#required-fields}
 
-在想要创建的 Kubernetes 对象所对应的 `.yaml` 文件中，需要配置的字段如下：
+在想要创建的 Kubernetes 对象所对应的 manifest（YAML 或 JSON 文件）中，需要配置的字段如下：
```
Comment (tengqm) [inline content/zh-cn/docs/concepts/overview/working-with-objects/_index.md:201]:
```suggestion
在想要创建的 Kubernetes 对象所对应的清单（YAML 或 JSON 文件）中，需要配置的字段如下：
```

**Comment by windsonsea** [comment]
Thanks.
/approve

**Pull request merged**